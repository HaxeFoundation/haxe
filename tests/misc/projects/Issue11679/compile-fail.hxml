-main Main
-D message.no-color
-D message.reporting=pretty
