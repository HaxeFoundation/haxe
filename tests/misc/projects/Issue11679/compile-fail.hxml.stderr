[ERROR] Macro.macro.hx:11: characters 4-9

 11 |    throw "Ohno";
    |    ^^^^^
    | Uncaught exception Ohno

     ->  Bar.hx:1: characters 1-8

     1 | @:build(Macro.build())
       | ^^^^^^^
       | Called from here

     ->  Macro.macro.hx:7: characters 24-33

     7 |     case TInst(cls,_): cls.get();
       |                        ^^^^^^^^^
       | Called from here

     ->  Main.hx:5: characters 1-8

     5 | @:build(Macro.build())
       | ^^^^^^^
       | Called from here

